### PR TITLE
Add git submodule for opentelemetry-proto, generate go files

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

Depends on https://github.com/signalfx/sapm-proto/pull/23

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -27,16 +27,64 @@ MISSPELL_CORRECTION=misspell -w
...
-generate:
-	mkdir -p gen
-	docker run --rm -v $(PWD):$(PWD) -w $(PWD) znly/protoc --gofast_out=./gen/ -I./ -I./vendor/github.com/gogo/protobuf/ -I./vendor/ sapm.proto
+# Target directory to write generated files to.
+PROTO_TARGET_GEN_DIR=./gen
+
+.PHONY: generate-sapm
+generate-sapm:
+	mkdir -p $(PROTO_TARGET_GEN_DIR)
+	docker run --rm -v $(PWD):$(PWD) -w $(PWD) znly/protoc --gofast_out=$(PROTO_TARGET_GEN_DIR)/ -I./ -I./vendor/github.com/gogo/protobuf/ -I./vendor/ sapm.proto
```
Comment (jgheewala) [inline makefile:41]: didn't you move sapm.proto to proto directory?

**Review by jgheewala — COMMENTED** [review]
Also do you want to pull in logs & metrics into sapm-proto? I do not think that's a good idea. sapm is meant only for trace my two cents would be not to pollute it with other proto's

File: makefile (hunk 1 above)
Comment (bogdandrutu) [reply makefile:41]: Independent PR. I will rebase when the other PR is merged

**Comment by bogdandrutu** [comment]
Sure will limit to generate only otlp trace

**Comment by bogdandrutu** [comment]
@jgheewala PTAL, only trace files are generated

**Review by jgheewala — APPROVED** [review]
lgtm

**Pull request merged**